src/views/components/cart.js

```diff
@@ -9,35 +9,22 @@ import {
   ImageBackground,
 } from 'react-native';
 import AsyncStorage from '@react-native-async-storage/async-storage';
-import img from '../utils/images';
-import storeData from '../services/store';
+import img from '../../utils/images';
+import {storeData} from '../../services/AsyncStorageService';
+import {clearStorage} from '../../services/AsyncStorageService';
 import {NavigationEvents} from 'react-navigation';
 
 export default class Cart extends Component {
   getData = async () => {
-    try {
-      const value = await AsyncStorage.getItem('save_data');
-      const jsonValue = value != null ? JSON.parse(value) : null;
-
-      console.log('data', jsonValue);
-      if (jsonValue !== null) {
-        this.setState({dataList: jsonValue});
-      }
-    } catch (e) {
-      console.log('error', e);
-    }
+    await (jsonValue = AsyncStorage.getItem('save_data')
```

Comment:
Move async storage-related operations to AsyncStorageService file

Reply:
done